tests/bofire/utils/test_torch_tools.py

```diff
@@ -858,6 +863,15 @@ def test_get_multiobjective_objective():
             "valid_omega": [1] * 10,
         },
     )
+
+    return samples, samples2, a_samples, obj1, obj2, obj3, obj4, experiments, outputs
+
+
+def test_get_multiplicative_objective(mutiobjective_data):
```

Comment:
Why did you rename this test? 
```suggestion
def test_get_multiobjective_objective(mutiobjective_data):
```